ss/ss.go

```diff
@@ -100,6 +100,12 @@ func toComDetails(ssOutput []string, protocol string, node *corev1.Node) ([]type
 		if err != nil {
 			return nil, err
 		}
+		name, err := identifyContainerForPort(debugPod, ssEntry)
```

Comment:
nit: it's not really clear what this function does (and the functions that it calls inside), e.g it says ForPort but receives an ssEntry. mind renaming them and/or adding some comments around them?

Reply:
I re-worked the naming of the functions and added comments. let me know if this is better now.